en/SUMMARY.md

```diff
@@ -29,21 +29,32 @@
   * [Interfacing to ROS](simulation/ros_interface.md)
   * [AirSim Simulation](simulation/airsim.md)
   * [Multi-Vehicle Simulation](simulation/multi-vehicle-simulation.md)
-* [Tutorials](tutorials/tutorials.md)
-  * [Ground Control Station](qgc/README.md)
-  * [Writing an Application](tutorials/tutorial_hello_sky.md)
-  * [Video Streaming in QGC](qgc/video_streaming.md)
-  * [Long-distance Video Streaming](qgc/video_streaming_wifi_broadcast.md)
+* [Hardware](hardware/README.md)
+  * [Autopilot Hardware](flight_controller/README.md)
+  * [Airframes](airframes/README.md)
+    * [Airframes Reference](airframes/airframe_reference.md)
+    * [Adding a New Airframe](airframes/adding_a_new_frame.md)
+    * [Parrot Bebop](advanced/parrot_bebop.md)
+  * [Porting Guide](debug/porting-guide.md)
+  * [Data Links](data_links/README.md)
+    * [SiK Radio](data_links/sik_radio.md)
+    * [Telemetry Wifi](advanced/telemetry_wifi.md)
+    * [Telemetry](tutorials/telemetry.md)
+  * [Sensor and Actuator Buses](sensor_bus/README.md)
+    * [I2C Bus](sensor_bus/i2c.md)
+      * [SF1XX lidar](sensor/sf1xx_lidar_setup.md)
```

Comment:
[This](https://hamishwillee.gitbooks.io/havdevguide/content/v/1c01c94cbbd1bfcfabe658092ebdda87e1574ed1/en/sensor/sf1xx_lidar_setup.html) will move to user guide - it isn't a developer topic.

Reply:
Done.